src/components/projects/Projects.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import './Projects.css';
+import floppyHat from '../../images/floppy-hat.png';
+import {
+  webDevProjects,
+  dataAnalysisProjects,
+  dataScienceProjects,
+} from '../../arrays/delegate-tasks';
+import Project from '../project/Project.js';
+
+export default function Projects({ title, subtitle, defaultDisplay, displayCourseButtons }) {
+  const [focusedCourse, setCourse] = React.useState(defaultDisplay);
+  const [displayedProjects, setProjects] = React.useState([]);
+  const [isExpanded, setExpanded] = React.useState(false);
+
+  React.useEffect(() => {
+    setCourse(webDevProjects);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  React.useEffect(() => {
+    setProjects(focusedCourse.slice(0, 2));
+  }, [focusedCourse]);
+
+  function updateCourse(course) {
+    setCourse(course);
+  }
+
+  function showMore() {
+    if (!isExpanded) {
+      setProjects(focusedCourse);
+    } else {
+      setProjects(focusedCourse.slice(0, 2));
```

Comment:
Great job on the show more button!

Since there is an unknown amount of projects in the cms, you can use a count variable for the second argument in your slice method

I did something similar here https://github.com/thinklikeadesigner/news-explorer-frontend/blob/main/src/components/CardsList/CardsList.js

Reply:
@thinklikeadesigner , very smart! this already looked amazing, but that's a good idea